src/auth/auth.ts

```diff
@@ -880,6 +884,8 @@ class AuthService {
 			email_verified: email_verified || false,
 			iat: Math.floor(Date.now() / 1000),
 			id,
+			login_address,

```

Comment:
these values `login_address` and `login_wallet` should only be assigned to the `tokenContent` if they are not null or undefined.